### PR TITLE
19th debriefing session

### PR DESCRIPTION
**Pull request by rgaiacs**
@k8hertweck, @raynamharris, @BillMills, @ttimbers Could you review this?

### CODE REVIEW

File: blog/2015/10/instructor-debriefing-2015-09-29.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<p>
+At one of the workshops instructors had problem due mix of files with the same
+name but different case, e.g. <pre>file.txt</pre> and <pre>File.txt</pre>.
+Windows and OS X are case insensitive systems!
+</p>
+<h2>Surveys</h2>
+<p>
+Some instructors suggested improvements to the pre- and post-workshop survey.
+The assessment subcommittee is working on a new version of the survey.
+<strong>If you want to help with the surveys please contact Jason Williams.</strong>
```
Comment (rgaiacs) [inline blog/2015/10/instructor-debriefing-2015-09-29.html]: @JasonJWilliamsNY This is OK for you?

File: blog/2015/10/instructor-debriefing-2015-09-29.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<blockquote>
+  it still caused a few pacing and conceptual problems,
+  with lesson threads (in particular the NumPy and matplotlib material)
+  getting a bit lost down digression/overly technical rabbit holes,
+  and the fact that some language basics are taught
+  after the more advanced use of libraries and data manipulation.
+</blockquote>
+</p>
+<p>
+The development subcommittee, in charge of our lessons,
```
Comment (rgaiacs) [inline blog/2015/10/instructor-debriefing-2015-09-29.html]: @jiffyclub This is OK for you?

File: blog/2015/10/instructor-debriefing-2015-09-29.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+had one meeting on October 15th where changes to the Python lesson
+were discuss. <strong>More information soon.</strong>
+</p>
+<h2>Live log</h2>
+<p>
+<a href="https://douglatornell.github.io/2015-09-22-ubc/">Last workshop at University of British Columbia</a>
+had a live log of shell commands during the shell and mercurial lessons on a web page.
+Doug allow us to share how they did it.
```
Comment (rgaiacs) [inline blog/2015/10/instructor-debriefing-2015-09-29.html]: I will add more information on this during the weekend.

File: blog/2015/10/instructor-debriefing-2015-09-29.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+</p>
+<h2>Steps after the workshop</h2>
+<p>
+Instructors from the University of British Columbia
+also reported that they have a community study group
```
Comment (rgaiacs) [inline blog/2015/10/instructor-debriefing-2015-09-29.html]: @douglatornell "community study group" is OK?

**Comment by rgaiacs** [comment]
The notes about this debriefing session are available on https://old.etherpad-mozilla.org/swc-debriefing-morning and https://old.etherpad-mozilla.org/swc-debriefing-evening.

**Comment by gvwilson** [comment]
Ready to merge?

**Comment by rgaiacs** [comment]
> Ready to merge?

@k8hertweck will add something to this soon. I will merge after that.

**Comment by k8hertweck** [comment]
Reviewed this one and nothing to add! Thanks.

**Comment by rgaiacs** [comment]
I will add the information about the live log. Forgot to do it yesterday.

File: blog/2015/10/instructor-debriefing-2015-09-29.html (hunk 4 above)
Comment (douglatornell) [inline blog/2015/10/instructor-debriefing-2015-09-29.html]: +1

**Comment by rgaiacs** [comment]
@gvwilson Sorry for my delay to add the missing part. Could you review and merge this?

**Comment by gvwilson** [comment]
Appears to have been merged in e4dc7db

**Pull request closed without merging**